docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -1,6 +1,280 @@
 ---
 title: Standardize access to observability data
-description: Learn how the Keptn Lifecycle Toolkit standardizes access to observability data.
+description: Learn how the Keptn Lifecycle Toolkit provides observability for Kubernetes deployments
 weight: 45
 ---
 
+The Keptn Lifecycle Toolkit (KLT) makes any Kubernetes deployment observable.
+You can readily see why a deployment takes so long or why it fails,
+even when using multiple deployment tools.
+Keptn introduces a concept of an application
+which is an abstraction that connects multiple
+Workloads belonging together.
+In other words, KLT, creates a distributed end-to-end trace
```

Comment:
I would move this sentence above, maybe also rephrase a bit: 

The Keptn Lifecycle Toolkit (KLT) makes any Kubernetes deployment observable; in other words, it creates a distributed end-to-end trace of everything Kubernetes does in the context of a Deployment.
The advantage is that you can readily see why a deployment takes so long or why it fails,
even when using multiple deployment tools. To do so, Keptn introduces the concept of an 'application'
which is an abstraction that connects multiple
Workloads belonging together.